worksheet.rb

```diff
@@ -31,4 +57,128 @@
 # - the total amount of money each driver has made
 # - the average rating for each driver
 # - Which driver made the most money?
-# - Which driver has the highest average rating?
\ No newline at end of file
+# - Which driver has the highest average rating?
+def rides_per_driver(rides)
+  dr_one_trips = 0
+  dr_two_trips = 0
+  dr_three_trips = 0
+  dr_four_trips = 0
+  
+  rides.each do |hash|
+    hash.each do |key, value|
+      if value == "DR0001"
+        dr_one_trips = dr_one_trips + 1
+      elsif value == "DR0002"
+        dr_two_trips = dr_two_trips + 1
+      elsif value == "DR0003"
+        dr_three_trips = dr_three_trips + 1
+      elsif value == "DR0004"
+        dr_four_trips = dr_four_trips + 1
+      end
+    end
+  end
```

Comment:
You should avoid creating separate variables for things like this.  Imagine what this would look like if you had to handle 100 drivers, or 1000, or all of the drivers that an actual company had.

Instead using an Array or Hash here would make things cleaner and more scalable.